addons/mail/models/mail_activity.py

```diff
@@ -603,6 +614,11 @@ class MailActivityMixin(models.AbstractModel):
         related='activity_ids.summary', readonly=False,
         search='_search_activity_summary',
         groups="base.group_user",)
+    decoration_type = fields.Selection([
```

Comment:
As we are in the activity mixin, these two fields must be prefixed with `activity_` (otherwise they could collide with existing fields).

Reply:
I would also use more explicit names, like `activity_exception_decoration` and `activity_exception_icon`.

Reply:
And again, I would like to see a comment explaining why we bypass ORM features for performance reasons on these fields.

Reply:
@mge-odoo Done